src/server/status_server.rs

```diff
@@ -238,8 +376,9 @@ impl StatusServer {
                         match (method, path.as_ref()) {
                             (Method::GET, "/metrics") => Box::new(ok(Response::new(dump().into()))),
                             (Method::GET, "/status") => Box::new(ok(Response::default())),
-                            (Method::GET, "/pprof/profile") => Self::dump_prof_to_resp(req),
+                            (Method::GET, "/debug/pprof/heap") => Self::dump_prof_to_resp(req),
                             (Method::GET, "/config") => Self::config_handler(config.clone()),
+                            (Method::GET, "/debug/pprof/profile") => Self::dump_rsperf_to_resp(req),
```

Comment:
Is `/debug/pprof/cpu` better?

Reply:
It follow go [`pprof`](https://golang.org/pkg/net/http/pprof/) url style

Reply:
OK